iree/base/time.c

```diff
@@ -33,6 +33,8 @@ IREE_API_EXPORT iree_time_t iree_time_now(void) {
   struct timespec clock_time;
   clock_gettime(CLOCK_REALTIME, &clock_time);
   return clock_time.tv_nsec;
+#elif defined(IREE_PLATFORM_GENERIC)
```

Comment:
you can move this to the top of the #if tree so that it's
```c
#if defined(IREE_TIME_NOW_FN)
IREE_TIME_NOW_FN
#elif .... 
```

That way it's possible to override on any platform that one may want (for example, if the user application already has its own timing stuff it wants to use).

Reply:
Done.